src/GitVersionCore/Common/BuildServerResolver.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using GitVersion.Logging;
+
+namespace GitVersion.Common
```

Comment:
Isn't everything within `GitVersionCore` "common"? What does the `Common` namespace add?

Reply:
GitVersionCore contains the versioning engine, the one that computes the version, as for the common folder, I moved some common interfaces/classes like FileSystem, Environment, BuildServer and so on. We can change that, any suggestions are welcomed.

Reply:
Yeah, I think `Common` is implied in the structure of the namespace. Anything that is within a namespace "parent" to the current one is to be considered "Common". So when you're in the  `GitVersion.SemanticVersioning.SemanticVersion` class for instance, everything within the `GitVersion.SemanticVersioning` as well as the `GitVersion` namespace is to be considered "Common" to your scope. C# even makes the `using` implicit in these cases, which I think underscores how this is inherent in the design of C# itself. So I think we can just remove the word `Common` altogether.

Reply:
Removed the `Common` from the namespace name